docs/pages/setup/admin/daemon.mdx

```diff
@@ -31,117 +35,174 @@ $ readlink /sbin/init
 
 </Notice>
 
-## Step 1/3. Install and configure Teleport
+## Install and configure Teleport
 
-Choose the appropriate instructions for your environment.
+If you have not already, select and complete the appropriate installation instructions for your environment.
 
 (!docs/pages/includes/install-linux.mdx!)
 
-Teleport requires a configuration file to run. After installation, no
-configuration file exists. We will create a minimal configuration file to show
-you how to run Teleport as a daemon:
+Teleport requires a [configuration yaml file](https://goteleport.com/docs/setup/reference/config/) to be created and configured in order to run. After initial installation, no
+configuration file exists until it is manually created by a user. This guide will use a minimal teleport configuration based on defaults, however in production environments we
+recommend [reviewing configuration file options](https://goteleport.com/docs/setup/reference/config/) to better suit your own specific needs.
+
+To create the default teleport configuration file, enter the following command:
 
 ```code
 $ sudo teleport configure -o file
 Wrote config to file "/etc/teleport.yaml". Now you can start the server. Happy Teleporting!
 ```
 
-## Step 2/3. Create a systemd unit file
+## Creating and Configuring the systemd Daemon
```

Comment:
@ptgott @rsyracuse let's keep the Step 2/3 structure for now. I plan to add a special plugin that will render this differently as well similar to Vault's docs to communicate that it's only 3 steps total and you are almost there (2/3)

<img width="245" alt="Screen Shot 2022-08-17 at 4 15 43 PM" src="https://user-images.githubusercontent.com/13717/185259787-20e8eb2f-1809-422a-99fa-eec9d5792f12.png">

Reply:
Re-added the steps, this makes a lot of sense